www/source/partials/docs/_using-builder-origin-keys.html.md.erb

```diff
@@ -0,0 +1,129 @@
+## <a name="origin-keys" id="origin keys" data-magellan-target="origin-keys">Using Origin Keys</a>
+
+## About origin keys
+
+Creating an origin automatically generates an origin key pair. 
+
+Origin key cryptography is asymmetric: it has a public origin key that you can distribute freely, and a private origin key that you should distribute only to members of the origin.
+
+Habitat Builder and Habitat Depot require access to at least the public origin key to validate packages in the origin. 
+
+Habitat uses the public origin key to verify the integrity of a downloaded Habitat artifact before installing it.
+
+By default, Habitat only installs packages for which ir has the public origin key.
+You can provide the public origin key to Habitat by pointing it to a depot that containing it, or by putting the public origin key on disk outside of Habitat in a location that Habitat knows to look for it, such as `~/.hab/cache/keys` on Linux systems or `C:\hab\cache\keys` on Windows systems.
+
+## Finding origin keys on Builder
+
+You can find your origin keys on the Builder site by navigating to the relevant origin and selecting the **Keys** tab.
+
+<img src="/images/screenshots/origin-keys.png">
+
+## Finding origin keys locally
+
+You'll find your saved private and public origin keys at `~/.hab/cache/keys` on your host(local) machine and  at `/hab/cache/keys` inside the studio environment. To see your existing origin keys, from your host command line, run:
+
+```shell
+ls -la ~/.hab/cache/keys
+```
+
+```powershell
+Get-ChildItem C:\hab\cache\keys
+```
+
+From within Habitat Studio, run:
+
+```shell
+ls -la ~/hab/cache/keys
+```
+
+```powershell
+Get-ChildItem C:\hab\cache\keys
+```
+
+### Creating origin keys
+
+Creating an origin key pair on on the Habitat Builder site automatically generates two keys, a private origin key and a public origin key. You will use the private origin key to sign your Habitat artifacts and Supervisors will use the public origin key to verify the integrity of these `.hart` files.
```

Comment:
```suggestion
Creating an origin key pair on on the Habitat Builder site automatically generates two keys, a private origin key and a public origin key. Habitat will use the private origin key to sign your Habitat artifacts when they are created and the public origin key to verify the integrity of artifacts for the origin when they are installed.
```
The supervisor does not do verification; that's the standard Habitat install process.